### ISSUE TITLE
How to add new Tag

### ISSUE THREAD

**Issue by engharb** [s9e/TextFormatter#100]
Hello,
I'm trying to add new Tags and render them to specific Html element but that does not work. And also I tried to customize i.e the Emoji size and I followed one of the examples but no way.

Could you please give me a concreted example how to do.

Here is the scenario:
```
$text = "[ATTACH=CONFIG]123[/ATTACH]"
```
I want to add ATTACH tag and after that rendering the text as 
```
<img .../>
```

Regards,

**Comment by JoshyPHP** [comment]
That's correct, you can load a bundle's configuration into the Configurator instance then modify it as you wish before creating a new parser and renderer that you can save elsewhere. I've edited your post to remove the calls to `getParser()` and `getRenderer()` since you don't need them and they were overwritten by `extract()`.
